docs/iris/example_code/General/polar_stereo.py

```diff
@@ -15,9 +15,17 @@
 
 
 def main():
-    # Enable a future option, to ensure that the grib load works the same way
-    # as in future Iris versions.
-    iris.FUTURE.strict_grib_load = True
+    # Enable a future option, to ensure that the grib load uses the mechanism
+    # intended for future Iris versions.
+    iris.FUTURE.external_grib_support = True
+
+    # What if anything is needed here for the first version of iris_grib ?
+    # - we don't actually know yet ...
+    # What we might need in future...
+    #    import iris_grib
+    #    iris_grib.FUTURE.strict_grib_load = True
```

Comment:
Oh no it isn't : [updated iris-grib#9, comment](https://github.com/SciTools/iris-grib/pull/9#issuecomment-219436588)